src/components/programs/create/whatwedo.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+import WhatWeDo1 from "@/public/create/whatwedo1.webp";
+import WhatWeDo2 from "@/public/create/whatwedo2.webp";
+
+const WhatWeDo = () => {
+  return (
+    <div className="mx-auto flex flex-col pb-[20vh] pt-[15vh]">
+      <div className="z-1 absolute h-full w-full opacity-70 blur-3xl">
+        <div className="absolute right-[80%] top-[10%] h-[500px] w-[500px] rounded-full bg-acm-blue-800" />
+        <div className="absolute right-[-15%] top-[60%] h-[500px] w-[500px] rounded-full bg-acm-blue-800" />
+      </div>
+      <div className="z-10 mx-auto grid w-5/6 grid-cols-2 grid-rows-2 items-stretch gap-x-14 gap-y-24">
```

Comment:
```suggestion
      <div className="z-10 mx-auto grid w-5/6 grid-cols-2 gap-x-14 gap-y-24">
```
these classes don't do anything